cmd/zq/main.go

```diff
@@ -1,154 +1,14 @@
 package main
 
 import (
-	"flag"
 	"fmt"
 	"os"
 
-	"github.com/brimdata/zed"
-	"github.com/brimdata/zed/cli"
-	"github.com/brimdata/zed/cli/inputflags"
-	"github.com/brimdata/zed/cli/outputflags"
-	"github.com/brimdata/zed/cli/procflags"
-	zedquery "github.com/brimdata/zed/cmd/zed/query"
-	"github.com/brimdata/zed/cmd/zed/root"
-	"github.com/brimdata/zed/driver"
-	"github.com/brimdata/zed/pkg/charm"
-	"github.com/brimdata/zed/pkg/rlimit"
-	"github.com/brimdata/zed/pkg/storage"
-	"github.com/brimdata/zed/zio"
+	"github.com/brimdata/zed/cli/zq"
```

Comment:
Why not just put the Command here if zq is the command is only used here?

Reply:
The point is to be able to do an install from either brimdata/zed or brimdata/zq and the pending repo brimdata/zq will have a main that looks the same as cmd/zq/main.go.